### ISSUE TITLE
Map is always reset to specific zoom level

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#9293]
Taken from https://github.com/cgeo/cgeo/issues/9270#issuecomment-716890164 reported by @MagpieFourtyTwo as a possible regression after fixing that issue:

Between subsequent calls of ``Live Map`` or single cache map, the zoom is always reset to min(last zoom; 100 m)

**Comment by moving-bits** [comment]
I'm not sure if it's related, but there is this line in `maps/CGeoMap` (line 206ff):

```
/**
 * if live map is enabled, this is the minimum zoom level, independent of the stored setting
 */
private static final int MIN_LIVEMAP_ZOOM = 12;
```

and later on (line 627ff):
```
/**
 * Set the zoom of the map. The zoom is restricted to a certain minimum in case of live map.
 */
private void setZoom(final int zoom) {
    mapView.getMapController().setZoom(mapOptions.isLiveEnabled ? Math.max(zoom, MIN_LIVEMAP_ZOOM) : zoom);
}
```

This explains why Google Maps (and old Mapsforge) sets a certain max zoom level - but that does not explain, why Mapsforge does this as well. I haven't found any direct connection yet, but if there is one, then the change regarding the "+/- 3" zoom level diff. may have triggered this.

**Comment by eddiemuc** [comment]
Possible duplicate: #9288

**Comment by Lineflyer** [comment]
Should be solved with #9288.
@MagPieFourtyTwo
Feel free to test and confirm. A fresh brewed nightly is available.

**Comment by MagpieFourtyTwo** [comment]
Yep, it's working - zoom is preserved again now. Thx!
